packages/core/src/lib/filter/index.ts

```diff
@@ -291,11 +292,14 @@ class Filter extends BaseProtocol implements IReceiver {
   }
 
   async createSubscription(
-    pubsubTopicShardInfo?: SingleShardInfo
+    pubsubTopicShardInfo?: SingleShardInfo | string
   ): Promise<Subscription> {
-    const pubsubTopic = pubsubTopicShardInfo
-      ? singleShardInfoToPubsubTopic(pubsubTopicShardInfo)
-      : DefaultPubsubTopic;
+    const pubsubTopic =
```

Comment:
wondering is `this.initializePubsubTopic` can be used here or adjusted to be used

Reply:
`this.initializePubsubTopic` is used to set the pubsub topics for the `Filter` object, while this `const pubsubTopic` is used to get/set the pubsub topic for the `Subscription` object.

Reply:
updated ternary expression to be less verbose in latest commit